### ISSUE TITLE
Style Friday

### ISSUE THREAD

**Issue by hani-q** [BugiDev/react-native-calendar-strip#170]
Hi,

Is there a way to style the text color of Friday and the date on that specific Friday ?

**Comment by peacechen** [comment]
The `customDatesStyles` prop supports that.
https://github.com/BugiDev/react-native-calendar-strip#customdatesstyles

It sounds like you'll need to use `dateNameStyle` and `dateNumberStyle`.

**Comment by hani-q** [comment]
dateNameStyle={{color:colors.veryLightGrey}}
      dateNumberStyle={{color:colors.veryLightGrey}}

already setting this to change style,, But how to target Friday specifically any examples ?

**Comment by peacechen** [comment]
Sorry I wasn't clear about those props.  Use the `customDatesStyles` prop which takes an array of objects.  The keys of each object specify the date and individual styles.

**Comment by hani-q** [comment]
Hi,

   {
                        startDate: new Date(), // Single date since no endDate provided
                        dateNameStyle: {color: 'blue'},
                        dateNumberStyle: {color: 'purple'},
                        // Random color...
                        dateContainerStyle: { backgroundColor: `#${(`#00000${(Math.random() * (1 << 24) | 0).toString(16)}`).slice(-6)}` },
                    }

ok Thanks,

but still stuck on How to mark each Friday only on the calender

**Comment by peacechen** [comment]
This feature will need to be added.  It could be added through a new prop, or enhance `customDatesStyles` to accept a callback function.  The callback would receive a date, and return a style if the date matches the user's preferences.  For example:
```
function styleFridays(date) {
  if (date.isoWeekday() === 5) {
    return {
      dateNameStyle: {color: 'blue'},
      dateNumberStyle: {color: 'purple'},
      dateContainerStyle:  {color: 'yellow'},
    }
  }
}

return (
  <CalendarStrip
    customDatesStyles={styleFridays}
  />
);
```

This is a community supported project and your contributions help yourself and others.  Please submit a PR and I will review & merge promptly.

**Comment by peacechen** [comment]
This feature has been released in 1.4.2.  See the Readme for an example passing a callback to customDatesStyles.
